graphql/mutation_resolver.go

```diff
@@ -513,14 +516,17 @@ func (r *mutationResolver) AttachProjectToNewRepo(ctx context.Context, project M
 }
 
 // AttachProjectToRepo is the resolver for the attachProjectToRepo field.
```

Comment:
I think most (if not all) of the Project Settings mutations are correctly receiving `projectId`s from the frontend. Do we want to start sending `projectIdentifier`s from the frontend? 

I guess I question this change because `projectId` (immutable) seems more resilient than `projectIdentifier` (usually not changed but not immutable). I agree with changing `identifier` -> `projectIdentifier` though.

Reply:
It's okay with me as long as all of the Project Settings transformers functions and Project Settings fields are updated to send the identifier instead of the ID 👍

Reply:
Thanks for this call out! I'm gonna revert much of these changes to still use `projectID` I read the frontend code some more, even though we reference `projectIdentifier` in much of the user-facing ways we still rely on `projectId` under the hood and I think its important we maintain this distinction.